packages/valory/services/counter/service.yaml

```diff
@@ -9,4 +9,9 @@ fingerprint:
 fingerprint_ignore_patterns: []
 number_of_agents: 1
 agent: valory/counter:0.1.0:bafybeidmoafileokvz4wrx2o4ry2sgtpngcvdn63pkmqswm5qjctnes6b4
-network: hardhat
+network:
+  id: hardhat
+  address: http://localhost:8545
+  chain_id: 31337
+  poa_chain: false
+  default_gas_price_strategy: eip1559
```

Comment:
These are already defined on the ledger connection, I am not sure why we are redefining this here...

Reply:
These are service level overrides, these variables will be exported as env variables in a docker container and picked up by the aea framework when building the aea.

Reply:
Then they should be specified as overrides for the underlying component on the relevant yaml page, same as we do for the agents imo.